specification/trace/semantic_conventions/instrumentation/elasticsearch.md

```diff
@@ -0,0 +1,64 @@
+# Semantic conventions for Elasticsearch
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply when creating a span for requests to Elasticsearch.
+
+## Span Name
```

Comment:
Minor non-blocking comment: an example could help people writing new instrumentation for Elastic, etc.

Reply:
This is ready to merge once this comment is acknowledged or addressed :)

Reply:
Example was added in cbdfd90c10c8527ca9120b8b5aa798cc9b317e33, this is ready to go :shipit: